site/.storybook/main.js

```diff
@@ -22,28 +21,38 @@ module.exports = {
         actions: false,
       },
     },
+    {
```

Comment:
on storybook@7 we now need to mannually import GFM markdown: https://storybook.js.org/docs/react/writing-docs/mdx#lack-of-github-flavored-markdown-gfm